docs/readmes/lte/build_install_magma_pkg_in_agw.md

```diff
@@ -74,3 +78,80 @@ hide_title: true
     ```bash
     apt show magma
     ```
+
+## Steps with Bazel
```

Comment:
Can we already replace "steps" with the new "steps with Bazel" section?

Reply:
The "steps" section is for 1.8 where this change is not supported.

Reply:
I know, I was suggesting to not keep that around anymore. So someone who looks up how to build a deb package today will find the Bazel version.

Reply:
done